custom_components/wyzeapi/wyzeapi/wyzeapi.py

```diff
@@ -56,26 +59,32 @@ def is_valid_login(self):
 			return False
 		return True
 
-	def list_bulbs(self):
-		url = "https://api.wyzecam.com/app/v2/home_page/get_object_list"
 
-		payload = {
-			"phone_system_type":"1",
-			"app_version":"2.6.62",
-			"app_ver":"com.hualai.WyzeCam___2.6.62",
-			"sc":"9f275790cab94a72bd206c8876429f3c",
-			"ts":"1575953834054",
-			"sv":"9d74946e652647e9b6c9d59326aef104",
-			"access_token": self._access_token,
-			"phone_id": self._device_id,
-			"app_name":"com.hualai.WyzeCam"
-		}
+	def get_devices(self):
+		if not self._all_devices:
```

Comment:
Since self._all_devices is initialized to [] in the __init__() function does this prevent the function from retrieving devices?

Reply:
@josephyancey

Reply:
In python [] is falsey. The first time it runs the list is empty so it actually gets the list of devices and saves it in _all_devices. If it needs to run again, the devices are already set so it doesn't go grab them again from the api.

Reply:
Great I have tested it and it works!